registry.json

```diff
@@ -1,5 +1,19 @@
 [
 
+   {
+       "ticker": "LOVE2",                                                       
+       "poolId": "65a6837264ea1d99e8af24e3a0f54cd4b3821919eb3e59b3a56b857e",  
+       "social": {              
+         "website":"https://love2stake.com", 
+         "twitter": "love2stake",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "love2stake",                                             
```

Comment:
Please can you add a user TG handle? This must be the handle of a user and not a group or channel.